packages/tenderly-utils/src/tenderly.ts

```diff
@@ -0,0 +1,126 @@
+import axios from 'axios'
+import { ethers, JsonRpcProvider } from 'ethers'
+import { IAccountGuardAbi, IAccountImplementationAbi } from './abis'
+import { NetworkName, TokenName, tokenAddresses } from './utils'
+import { getTokenDecimals } from './getTokenDecimals'
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const { TENDERLY_USER, TENDERLY_PROJECT, TENDERLY_ACCESS_KEY } = process.env as any
+
+if (!TENDERLY_USER || !TENDERLY_PROJECT || !TENDERLY_ACCESS_KEY) {
+  throw new Error('Tenderly environment variables not set')
+}
+
+const TENDERLY_FORK_API = `https://api.tenderly.co/api/v1/account/${TENDERLY_USER}/project/${TENDERLY_PROJECT}/fork`
+
+const request = axios.create({
+  baseURL: 'https://api.tenderly.co/api/v1',
+  headers: {
+    'X-Access-Key': TENDERLY_ACCESS_KEY,
+    'Content-Type': 'application/json',
+  },
+})
+
+export const createFork = async ({
```

Comment:
Why did we decide not to go with a DevNet here?

Reply:
It was just faster to implement and does the job. Do you have any specific concerns that we are missing something here?

Reply:
Only that we need to remove forks after the fact.

Reply:
Agreed, we can have a look later to see if we can also quickly add devnets helper on top of what we already have. But should be fine for the initial work to not block the development.

Reply:
Forks can be snapshotted, and revert back after each test. So it's not longer a caveat.